hw/sch/hw_releases.md

```diff
@@ -60,15 +60,13 @@ that acts as a pass-through for audio and control signals.
 Up to 5 Expansion Units can be added to one Controller.
```

Comment:
Shouldn't these be called zone expanders?

Reply:
You know, I totally had "expansion unit" as the official name in my head but you're right!